chelsea-shayne/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Lab 1</title>
+  <meta name="viewport" content="width=device-width" />
```

Comment:
You forgot to set the initial scale, it should look like this

`<meta name="viewport" content="width=device-width, initial-scale=1">`

Also next time leave the TODO comment here and instead of deleting it, change TODO to DONE :)